auth-web/src/components/auth/staff/account-management/StaffActiveAccountsTable.vue

```diff
@@ -182,244 +182,252 @@
+                value?.orgType === org.orgType
+      )
+      if (byAccessTypeAndOrgType) {
+        return byAccessTypeAndOrgType[0]
+      }
+      const byAccessType = entries.find(([, value]) =>
+        value?.accessType?.includes(org.accessType)
+      )
+      if (byAccessType) {
+        return byAccessType[0]
+      }
+      const byOrgType = entries.find(([, value]) =>
+        value?.orgType === org.orgType
+      )
+      if (byOrgType) {
+        return byOrgType[0]
+      }
+      return ''
+    }
 
-  // Used to go from 'Basic' -> accessType: AccessType.REGULAR, orgType: Account.BASIC
-  protected getOrgAndAccessTypeFromAccountType (accountType: string): object {
-    return this.accountTypeMap[accountType]
-  }
+    const noDataMessage = computed(() => {
+      return t(
+        state.searchParamsExist
+          ? 'searchAccountNoResult'
+          : 'searchAccountStartMessage'
+      )
+    })
 
-  // Used to go from OrgType -> OrgAccountTypes
-  protected getAccountTypeFromOrgAndAccessType (org:Organization): any {
-    const entries = Object.entries(this.accountTypeMap)
-    const byAccessTypeAndOrgType = entries.find(([, value]) =>
-      value?.accessType?.includes(org.accessType) &&
-                                                  value?.orgType === org.orgType)
-    if (byAccessTypeAndOrgType) {
-      return byAccessTypeAndOrgType[0]
-    }
-    const byAccessType = entries.find(([, value]) =>
-      value?.accessType?.includes(org.accessType))
-    if (byAccessType) {
-      return byAccessType[0]
+    function setSearchFilterToStorage (val: string): void {
+      ConfigHelper.addToSession(SessionStorageKeys.OrgSearchFilter, val)
     }
-    const byOrgType = entries.find(([, value]) =>
-      value?.orgType === org.orgType)
-    if (byOrgType) {
-      return byOrgType[0]
-    }
-    return ''
-  }
 
-  protected get noDataMessage () {
-    return this.$t(
-      this.searchParamsExist
-        ? 'searchAccountNoResult'
-        : 'searchAccountStartMessage'
-    )
-  }
-
-  protected setSearchFilterToStorage (val:string):void {
-    ConfigHelper.addToSession(SessionStorageKeys.OrgSearchFilter, val)
-  }
+    function doSearchParametersExist (params: OrgFilterParams): boolean {
+      return params.name.length > 0 ||
+                params.branchName.length > 0 ||
+                params.id.length > 0 ||
+                params.bcolAccountId.length > 0 ||
+                params.decisionMadeBy.length > 0 ||
+                (params.orgType.length > 0 && params.orgType !== OrgAccountTypes.ALL)
+    }
 
-  protected doSearchParametersExist (searchParams: OrgFilterParams) {
-    return searchParams.name.length > 0 ||
-          searchParams.branchName.length > 0 ||
-          searchParams.id.length > 0 ||
-          searchParams.decisionMadeBy.length > 0 ||
-          (searchParams.orgType.length > 0 && searchParams.orgType !== OrgAccountTypes.ALL)
+    mounted()
```

Comment:
Looks like the difference here between setup and onMounted... 
onMounted has access to the DOMand setup doesn't.